packages/publikator/graphql/resolvers/_mutations/commit.js

```diff
@@ -111,6 +111,14 @@ module.exports = async (_, args, context) => {
   // get / create repo
   let repo = await getRepo(repoId).catch((response) => null)
 
+  if (isTemplate && !mdast.meta.title) {
```

Comment:
As far as I see, this only concerns new documents. Wouldn't it make sense to move these checks into the `else` clause, just before `repo = await githubRest.repos.createInOrg()`.

Reply:
Maybe we could start to refactor this fuction a wee bit?

Something along the lines of

```js
const repo = createOrGetRepo(...)
```

Reply:
No it's also on existing repos. That's why I put this outside of the if-else. 
Basically if a repo is a so-called template repo, it must have a title and a slug in its metadata. Otherwise a user could create a repo (with title/slug), then remove either of those params in a subsequent commit without it raising an error.